templates/confidenceSlider.jsx

```diff
@@ -69,6 +69,7 @@ export default function ConfidenceSlider (props) {
         ])}
         aria-labelledby={ariaQuestion ? null : (displayTitle || body || instruction) && `${_id}-header`}
         aria-label={ariaQuestion || null}
+        role='region'
```

Comment:
Why [region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/group_role) and not [group](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/group_role)?

![image](https://github.com/user-attachments/assets/448935be-881d-4204-9199-5cf7701a2e10)

![image](https://github.com/user-attachments/assets/e23d2c9b-1043-428c-8ae6-d20bcd1ad0aa)

Reply:
Group has been used where there is a collection of items with the same functionality MCQ/textInput etc, I felt region was better suited as it's not a collection of items but still important and we do want users to navigate to the section easily.

Reply:
Group is not explicitly for a collection of items with the same functionality, it's not like `role=list`. Group is more for a bounded section of interactive user interface. Group is more akin to the grouping of an "Address" or "Card details" section on an order form.

Region sections are more [landmark](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/landmark_role) orientated, they have their own shortcut key. Using region on some widgets and group on others will have unintended consequences with keyboard navigation when using a screen reader.

Reply:
Thanks for the info, I shall update to group to avoid any issues with keyboard nav.